File: src/main/java/net/noyark/scpslserver/jsmod2/network/AdminQueryPacket.java
package net.noyark.scpslserver.jsmod2.network;

/**
 * @author magiclu550
 */

public class AdminQueryPacket extends DataPacket {

    public AdminQueryPacket(){
        super(0x01);
    }

    @Override
    public byte[] encode() {
        return new byte[0];
    }

    @Override
    public Object decode(byte[] bytes) {
        return null;
    }
}

File: proxy/scpDataNetwork/network/AdminQueryPacket.cs
using System;
using System.Collections.Generic;
using Smod2.Events;

namespace scpDataNetwork.network
{
    public class AdminQueryPacket : DataPacket
    {
        
        private AdminQueryEvent e;

        public AdminQueryPacket() : base(0x01)
        {
            
        }

        public override byte[] encode()
        {
            return dataObjectEncode(e);
        }

        public override object decode(byte[] datas)
        {
            return dataObjectDecode(datas,typeof(AdminQueryEvent));
        }
    }
}